### DISCUSSION TITLE
following bridged account from a bridged account

### DISCUSSION THREAD

**Discussion by lucajet** [snarfed/bridgy-fed#1555] [Q&A]
With a mastodon-bridged account, on BlueSky are listed my 'native' followers from BlueSky.
And i'm listed too as some others BlueSky follower.

Now let's say i follow another native account on mastodon, and we are both bridged to BS.
Shouldn't that relation be listed on BlueSky as well ?

(i know it's not, but would be nice)

**Comment by snarfed** [comment]
Yes! Great question. We do this kind of thing as much as we can. Eg when a bridged fedi user replies to another bridged fedi user, if we get delivered the reply, we bridge it. We'd happily do this with followers and other interactions too!...but they're generally not bridged to us, so we don't see them.

We could proactively fetch everyone's followers proactively and try to sync and bridge them. Low priority, but definitely possible.

**Comment by lucajet** [comment]
instead of fetching all those data periodically, that look expensive, maybe with a DM to the bridge from one of the actor .

Something like

"look, i follow my_dear_friend@bsky.brid.gy"

you can then reverse back the name and check if it's true on fediverse for both side, or just from one.
